tests/istanbul/test_state_transition.py

```diff
@@ -0,0 +1,135 @@
+from functools import partial
+from typing import Dict
+
+import pytest
+
+from ethereum.exceptions import InvalidBlock
+from tests.helpers.load_state_tests import (
+    Load,
+    fetch_state_test_files,
+    idfn,
+    run_blockchain_st_test,
+)
+
+fetch_istanbul_tests = partial(fetch_state_test_files, network="Istanbul")
+
+FIXTURES_LOADER = Load("Istanbul", "istanbul")
+
+run_istanbul_blockchain_st_tests = partial(
+    run_blockchain_st_test, load=FIXTURES_LOADER
+)
+
+# Run legacy general state tests
+test_dir = "tests/fixtures/BlockchainTests/GeneralStateTests/"
+
+# Every test below takes more than  60s to run and
+# hence they've been marked as slow
+SLOW_TESTS = (
+    "stTimeConsuming/CALLBlake2f_MaxRounds.json",
+    "stTimeConsuming/static_Call50000_sha256.json",
+    "vmPerformance/loopExp.json",
+    "vmPerformance/loopMul.json",
+    "QuadraticComplexitySolidity_CallDataCopy_d0g1v0_Istanbul",
+    "CALLBlake2f_d9g0v0_Istanbul",
+    "CALLCODEBlake2f_d9g0v0",
```

Comment:
Just wanted to confirm that these five tests are, in fact, slow? The first three sound slow, not sure about the last two.

Reply:
Yes. If I remember correctly (I had checked this way back when I raised the PR :)), the last two tests take about 6-7 mins on my local machine. So I guess they'll take even longer on CI.